Merge Master

wyyalt opened the pull request.

guohelu committed:
fix: 修改提示信息 --story=126514126
# Reviewed, transaction id: 58183

dengyh committed:
Merge pull request #8055 from guohelu/master_0917

fix: 修改配置错误提示信息 --story=126514126

dengyh committed:
feat: 更新版本号 --story=147958653
# Reviewed, transaction id: 58220

dengyh committed:
Merge pull request #8056 from dengyh/tag_3_32_1_p11

feat: 更新版本号 --story=147958653

dengyh committed:
Merge pull request #8023 from guohelu/master_250829

feat: 告警屏蔽插件支持自定义屏蔽原因 --story=126416676

dengyh committed:
Merge pull request #8034 from guohelu/master_0905

feat: 增加获取任务操作记录接口 --story=127075667

Mianhuatang8 committed:
fix: 转译以及设置校验状态 --story=127502680 (#8057)

* fix: 转译以及设置校验状态 --story=127502680
# Reviewed, transaction id: 58270

* fix: 转译以及设置校验状态 --story=127502680
# Reviewed, transaction id: 58272

* fix: 转译以及设置校验状态 --story=127502680
# Reviewed, transaction id: 58273

* fix: 调换校验先后顺序 --story=127502680
# Reviewed, transaction id: 58277

dengyh committed:
feat: 更新版本号 --story=147958653
# Reviewed, transaction id: 58299

dengyh committed:
Merge pull request #8058 from dengyh/release_3_32_2

feat: 更新版本号 --story=147958653

guohelu committed:
fix: 修改返回信息 --story=127075667
# Reviewed, transaction id: 58332

dengyh committed:
Merge pull request #8059 from guohelu/master_250917

fix: 修改返回信息 --story=127075667

Mianhuatang8 committed:
fix: 转译以及tooltips --story=127502680
# Reviewed, transaction id: 58360

dengyh committed:
feat: 更新版本号到3.32.7 --story=147958653
# Reviewed, transaction id: 58400

Mianhuatang8 committed:
fix: 添加失败回调记录 #ignore
# Reviewed, transaction id: 58457

dengyh committed:
fix: 修复fast_create_task生成的任务获取任务详情报错的问题 --bug=148276076
# Reviewed, transaction id: 58527

dengyh committed:
Merge pull request #8063 from dengyh/fix/fast_task_detail

fix: 修复fast_create_task生成的任务获取任务详情报错的问题 --bug=148276076

guohelu committed:
fix: 修复参数兼容性问题 --story=127611598
# Reviewed, transaction id: 58885

guohelu committed:
feat: 解除屏蔽支持多个策略ID --story=127337064
# Reviewed, transaction id: 58898

guohelu committed:
fix: 修复插件单侧失败问题 --story=127337064
# Reviewed, transaction id: 58904

guohelu committed:
fix: 修复wenhook参数序列化错误 --story=147958653
# Reviewed, transaction id: 58911

guohelu committed:
fix: 修复wenhook回调参数问题 --story=127611598
# Reviewed, transaction id: 58912

dengyh committed:
Merge pull request #8066 from guohelu/master_0923

fix: 修复wenhook参数序列化错误 --story=147958653

dengyh committed:
Merge pull request #8065 from guohelu/master_0922

feat: 解除屏蔽支持多个策略ID --story=127337064

dengyh committed:
feat: 更新版本 --story=127337064
# Reviewed, transaction id: 59118

dengyh committed:
Merge pull request #8067 from dengyh/feat/v_3_32_8

feat: 更新版本 --story=127337064

guohelu committed:
fix: 修复参数类型处理错误 --story=127337064
# Reviewed, transaction id: 59170

dengyh committed:
Merge pull request #8068 from guohelu/master_0925

fix: 修复参数类型处理错误 --story=127337064

guohelu committed:
fix: 修复循环流导致yaml导出失败 --story=148536703
# Reviewed, transaction id: 59565

guohelu committed:
fix: 调整参数处理逻辑 --story=127611598
# Reviewed, transaction id: 59677

guohelu committed:
fix: 序列化器添加默认值 --story=127611598
# Reviewed, transaction id: 59685

guohelu committed:
fix: 修复代码导包规范问题 --story=127611598
# Reviewed, transaction id: 60047

dengyh committed:
Merge pull request #8064 from guohelu/master_0920

fix: 修复参数兼容性问题 --story=127611598

guohelu committed:
fix: 修复资源审批人为创建人和更新人 --story=127854750
# Reviewed, transaction id: 60395

dengyh committed:
Merge pull request #8069 from guohelu/master_0929

fix: 修复循环流导致yaml导出失败 --story=148536703

guohelu committed:
fix: 增加去重逻辑 --story=127854750
# Reviewed, transaction id: 60507

guohelu committed:
fix: 修复任务子流程未执行节点变量渲染失败 --story=149102381
# Reviewed, transaction id: 60517

dengyh committed:
Merge pull request #8072 from guohelu/master_1014

fix: 修复任务子流程未执行节点变量渲染失败 --story=149102381

dengyh committed:
Merge pull request #8071 from guohelu/master_1010

fix: 修复资源审批人为创建人和更新人 --story=127854750

guohelu committed:
fix: 兼容流程通知方式默认数据 --story=148213756
# Reviewed, transaction id: 60636

dengyh committed:
Merge pull request #8073 from guohelu/master_1015

fix: 兼容流程通知方式默认数据 --story=148213756

dengyh committed:
refactor: 修改版本日志 --story=148536703

guohelu committed:
fix: 列表接口兼容流程通知方式默认数据 --story=148213756
# Reviewed, transaction id: 60646

dengyh committed:
Merge pull request #8074 from guohelu/master_15

fix: 列表接口兼容流程通知方式默认数据 --story=148213756

dengyh committed:
refactor: 支持通过配置来定制化不同项目的任务清理需求 --story=128038083

dengyh committed:
refactor: 自动化code review --story=128038083

dengyh committed:
refactor: 自动化code review --story=128038083

dengyh committed:
refactor: 自动化code review --story=128038083

dengyh committed:
refactor: 优化清理逻辑性能 --story=128038083

dengyh committed:
refactor: 优化清理逻辑性能 --story=128038083

dengyh committed:
Merge pull request #8078 from dengyh/refactor/special_clean

refactor: 支持通过配置来定制化不同项目的任务清理需求 --story=128038083

dengyh committed:
refactor: celery worker增加自动重启的逻辑 --story=128084168

dengyh committed:
Merge pull request #8079 from dengyh/refactor/worker_auto_restart

refactor: celery worker增加自动重启的逻辑 --story=128084168

dengyh committed:
refactor: 优化错误码处理逻辑 --story=128084168

dengyh committed:
Merge pull request #8080 from dengyh/refactor/worker_auto_restart

refactor: 优化错误码处理逻辑 --story=128084168

Mianhuatang8 committed:
fix: 回调记录请求结果兼容接口不同数据形式 --story=128219322
# Reviewed, transaction id: 61995

dengyh committed:
feat: 修复模板变量统计任务重复执行的问题 --story=149770577

dengyh committed:
feat: 优化清理任务的逻辑 --story=149770577

dengyh committed:
Merge pull request #8085 from dengyh/refactor/worker_auto_restart

Refactor/worker auto restart

Mianhuatang8 committed:
fix: 公共流程编辑无法保存 --story=128312632
# Reviewed, transaction id: 62482

guohelu committed:
fix: 修复模板导出和周期任务更新问题 --story=149963760
# Reviewed, transaction id: 62942

dengyh committed:
refactor: 支持项目自定义展示过滤 --story=128038083

dengyh committed:
Merge pull request #8088 from dengyh/refactor/special_clean

Refactor/special clean

dengyh committed:
refactor: 所有celery worker默认增加自动重启的机制 --story=128084168

dengyh committed:
refactor: 防止cursor超限制导致CI报错 --story=128084168

dengyh committed:
Merge pull request #8089 from dengyh/refactor/worker_auto_restart

refactor: 所有celery worker默认增加自动重启的机制 --story=128084168

guohelu committed:
feat: http回调增加参数 --story=128420474
# Reviewed, transaction id: 63377

Mianhuatang8 committed:
fix: 回调创建请求过滤请求头key为空的数据 --story=128484082
# Reviewed, transaction id: 63556

Mianhuatang8 committed:
fix: 因空格编码不同导致搜索流程名无结果 --story=126793365 (#8014)

guohelu committed:
feat: 企微插件新增版本支持markdown_v2格式 --story=128614608
# Reviewed, transaction id: 64462

guohelu committed:
fix: 修改插件提示信息 --story=128614608

dengyh committed:
refactor: 改成使用codebuddy来进行code review --story=127611598

dengyh committed:
Merge pull request #8092 from guohelu/master_1113

feat: 企微插件新增版本支持markdown_v2格式 --story=128614608

dengyh committed:
Merge pull request #8087 from guohelu/master_1031

fix: 修复模板导出和周期任务更新问题 --story=149963760

dengyh committed:
Merge pull request #8090 from guohelu/master_1105

feat: HTTP回调增加参数 --story=128420474

dengyh committed:
refactor: celery prometheus exporter 增加重启机制 --story=128084168

dengyh committed:
Merge pull request #8093 from dengyh/refactor/prom_celery_auto_restart

refactor: celery prometheus exporter 增加重启机制 --story=128084168

guohelu committed:
feat: 企微插件新增版本 --story=128614608
# Reviewed, transaction id: 64899

guohelu committed:
fix: 修复插件提示信息 --story=128614608
# Reviewed, transaction id: 64902

dengyh committed:
Merge pull request #8094 from guohelu/master_1117

feat: 企微插件新增版本 --story=128614608

dengyh committed:
feat: 更新版本号 --story=128614608

Mianhuatang8 committed:
fix: 关闭url校验 #ignore
# Reviewed, transaction id: 65286

dengyh committed:
refactor: callback接口支持异步重试来作为降级处理的方案 --story=128811166

dengyh committed:
Merge pull request #8097 from dengyh/refactor/callback_optimize

refactor: callback接口支持异步重试来作为降级处理的方案 --story=128811166

dengyh committed:
refactor: 增加日志排查callback失败的问题 --story=128811166

dengyh committed:
Merge pull request #8098 from dengyh/refactor/callback_optimize

refactor: 增加日志排查callback失败的问题 --story=128811166

guohelu committed:
feat: http回调提供批量更新接口 --story=128443156
# Reviewed, transaction id: 66148

guohelu committed:
fix: 修改代码规范和逻辑 --story=128443156
# Reviewed, transaction id: 66155

dengyh committed:
refactor: 优化同步重试的逻辑 --story=128811166

dengyh committed:
Merge pull request #8100 from dengyh/refactor/callback_optimize

refactor: 优化同步重试的逻辑 --story=128811166

Mianhuatang8 committed:
fix: 新建任务回退跳转问题修复 --story=127740619
# Reviewed, transaction id: 66326

dengyh committed:
feat: job部分内置插件支持host_id作为服务器参数 --story=129036114

dengyh committed:
Merge pull request #8103 from dengyh/feat/job_support_hostid

feat: job部分内置插件支持host_id作为服务器参数 --story=129036114

guohelu committed:
fix: 优化处理性能问题 --story=128443156
# Reviewed, transaction id: 67234

guohelu committed:
fix: 补充配置参数的校验 --story=128443156
# Reviewed, transaction id: 67312

dengyh committed:
Merge pull request #8099 from guohelu/master_1125

feat: HTTP回调提供批量更新接口 --story=128443156

forrany committed:
feat: update Babel configuration and dependencies to support new JavaScript features and ai-blueking

Commit:
refactor: 添加获取标准运维智能体服务地址的环境变量逻辑--story=129317277

dengyh committed:
Merge pull request #8114 from kemafd1sf/add_sops_agent_server_url_env

refactor: 添加获取标准运维智能体服务地址的环境变量逻辑--story=129317277

tencentblueking-adm commented:
[![CLA assistant check](https://bk-cla.bktencent.com:443/pull/badge/not_signed)](https://bk-cla.bktencent.com:443/TencentBlueKing/bk-sops?pullRequest=7844) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://bk-cla.bktencent.com:443/TencentBlueKing/bk-sops?pullRequest=7844) before we can accept your contribution.<br/>**3** out of **4** committers have signed the CLA.<br/><br/>:white_check_mark: dengyh<br/>:white_check_mark: RGardenia<br/>:white_check_mark: Mianhuatang8<br/>:x: kemafd1sf<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://bk-cla.bktencent.com:443/check/TencentBlueKing/bk-sops?pullRequest=7844) it.</sub>

Commit:
refactor: 前端代码添加获取智能体服务地址的环境变量获取逻辑--story=129317277

Mianhuatang8 committed:
fix: 新建任务变量隐藏只有最后一个触发条件生效 --story=129420191
# Reviewed, transaction id: 68660

dengyh committed:
Merge pull request #8115 from kemafd1sf/add_sops_agent_server_url_env

refactor: 前端代码添加获取智能体服务地址的环境变量获取逻辑--story=129317277

Mianhuatang8 committed:
feat: 接入ai小鲸 --story=129317277 (#8118)

# Reviewed, transaction id: 68678

dengyh committed:
feat: get_task_lsit 接口支持通过时间和创建方式来进行过滤 --story=129127388

dengyh committed:
Merge pull request #8119 from dengyh/feat/get_task_list

feat: get_task_lsit 接口支持通过时间和创建方式来进行过滤 --story=129127388

Mianhuatang8 committed:
fix: 任务入参修改不允许修改配置为“常量”的变量 --story=129218796
# Reviewed, transaction id: 68712

dengyh committed:
feat: 网关MCP Server对返回的信息进行过滤 --story=129445652

dengyh committed:
Merge pull request #8121 from dengyh/feat/mcp_optimize

feat: 网关MCP Server对返回的信息进行过滤 --story=129445652

Mianhuatang8 committed:
feat: 公共流程可见范围增加复制粘贴按钮 --story=128274215
# Reviewed, transaction id: 68946

guohelu committed:
feat: 模板信息返回添加备注信息 --story=128443156
# Reviewed, transaction id: 69186

guohelu committed:
fix: 修复单侧失败错误 --story=128443156
# Reviewed, transaction id: 69190

dengyh committed:
feat: 增加接口获取任务执行的有效时长 --story=129548331

dengyh committed:
Merge pull request #8124 from dengyh/feat/task_effective_time

feat: 增加接口获取任务执行的有效时长 --story=129548331

dengyh committed:
Merge pull request #8123 from guohelu/master_1223

feat: 模板信息返回添加备注信息 --story=128443156

Mianhuatang8 committed:
fix: 新建任务第二步情况下跳转错误修复 #ignore
# Reviewed, transaction id: 69393

Mianhuatang8 committed:
fix: 回调请求头信息被隐藏问题修复 --story=128484082
# Reviewed, transaction id: 69431

dengyh committed:
feat: 增加接口获取任务执行的有效时长 --story=129548331

dengyh committed:
feat: 增加接口获取任务执行的有效时长 --story=129548331

dengyh committed:
Merge pull request #8128 from dengyh/feat/task_effective_time

feat: 增加接口获取任务执行的有效时长 --story=129548331

dengyh committed:
feat: 更新版本号 --story=129548331

guohelu committed:
fix: 修复网关api请求变量结构转换问题 --story=152536452
# Reviewed, transaction id: 69902

guohelu committed:
fix: 修复测试用例失败问题 --story=152536452
# Reviewed, transaction id: 70032

dengyh committed:
Merge pull request #8130 from guohelu/master_1229

fix: 修复网关api请求变量结构转换问题 --story=152536452

dengyh committed:
feat: 网关MCP Server对返回的信息进行过滤 --story=129445652

dengyh committed:
Merge pull request #8132 from dengyh/feat/mcp_optimize

feat: 网关MCP Server对返回的信息进行过滤 --story=129445652

dengyh committed:
feat: 增加判断是否来源于MCP的逻辑 --story=129445652

dengyh committed:
Merge pull request #8133 from dengyh/feat/mcp_optimize

feat: 增加判断是否来源于MCP的逻辑 --story=129445652

dengyh committed:
feat: 增加判断是否来源于MCP的逻辑 --story=129445652

dengyh committed:
Merge pull request #8134 from dengyh/feat/mcp_optimize

feat: 增加判断是否来源于MCP的逻辑 --story=129445652

Mianhuatang8 committed:
fix: TagSelect组件使用远程数据源时，请求支持携带cookie凭证信息 --story=129798291
# Reviewed, transaction id: 70745

Mianhuatang8 committed:
feat: 添加ai小鲸动态上下文 --story=129317277
# Reviewed, transaction id: 70836

Commit:
feat: apigw包下添加get_task_node_log和get_task_plugin_log两个接口--story=130004643

Mianhuatang8 committed:
fix: 正则校验问题修复 --story=129973255
# Reviewed, transaction id: 71286

dengyh committed:
Merge pull request #8144 from kemafd1sf/add_node_log_api_for_apigw

feat: apigw包下添加get_task_node_log和get_task_plugin_log两个接口--story=130004643

Commit:
fix: 修复apigw下两个接口get_task_node_log和get_task_plugin_log两个接口鉴权问题--story=130004643

dengyh committed:
Merge pull request #8147 from kemafd1sf/fix_node_log_api_for_apigw

fix: 修复apigw下两个接口get_task_node_log和get_task_plugin_log两个接口鉴权问题--story=130004643

dengyh committed:
refactor: 清理weixin目录 --story=130071508

luofann committed:
refactor: 升级node-forge版本 #ignore

Mianhuatang8 committed:
feat: 代码框表单增加AI工具 --story=130106980 (#8148)

dengyh committed:
refactor: 更新版本号 --story=130071508

dengyh committed:
Merge pull request #8150 from dengyh/refactor/remove_weixin

refactor: 清理weixin目录 --story=130071508

dengyh committed:
refactor: 优化api-resources.yml中网关API的声明 --story=130158303

dengyh committed:
Merge pull request #8152 from dengyh/refactor/api-resources-update

refactor: 优化api-resources.yml中网关API的声明 --story=130158303

RGardenia committed:
fix: 代码框表单增加AI工具 --story=130162239

Mianhuatang8 committed:
fix: 代码框脚本检查自动触发快捷指令 --story=130162239
# Reviewed, transaction id: 72225

Mianhuatang8 committed:
feat: 支持AI排查流程执行失败问题 --story=130202440
# Reviewed, transaction id: 72281

Mianhuatang8 committed:
feat: 支持AI排查流程执行失败问题 --story=130202440
# Reviewed, transaction id: 72282

Mianhuatang8 committed:
fix: 失效样式问题修复 --story=130202440
# Reviewed, transaction id: 72359

guohelu committed:
perf: 监控告警屏蔽相关插件优化 --story=129745657
# Reviewed, transaction id: 72639

Commit:
feat: 添加获取任务有效时长的的内部接口 --story=130531087

dengyh committed:
Merge pull request #8171 from kemafd1sf/feat/add_get_effective_time_inner

feat: 添加获取任务有效时长的的内部接口 --story=130531087

guohelu committed:
fix: 修复webhook请求参数json格式转换问题 --story=154050993
# Reviewed, transaction id: 73136

dengyh committed:
Merge pull request #8172 from guohelu/master_0130

fix: 修复webhook请求参数json格式转换问题 --story=154050993

dengyh committed:
feat: 构建节点执行的Span --story=130572156

dengyh committed:
refactor: 修复import的问题 --story=130255505

dengyh committed:
Merge pull request #8166 from guohelu/master_0126

perf: 监控告警屏蔽相关插件优化 --story=129745657

Mianhuatang8 committed:
feat: 支持AI进行任务总结 --story=130722324
# Reviewed, transaction id: 73326

dengyh committed:
Merge pull request #8174 from dengyh/feat/plugin_trace

feat: 构建节点执行的Span --story=130572156

dengyh committed:
refactor: 修复python版本兼容的问题 --story=130572156

dengyh committed:
Merge pull request #8177 from dengyh/feat/plugin_trace

refactor: 修复python版本兼容的问题 --story=130572156

dengyh committed:
refactor: plugin execute和schedule 的span名称格式修改 --story=130572156

dengyh committed:
Merge pull request #8178 from dengyh/feat/plugin_trace

refactor: plugin execute和schedule 的span名称格式修改 --story=130572156

Mianhuatang8 committed:
fix: 屏蔽告警插件策略搜索时无法多选问题修复 --story=130827380 (#8181)

* fix: 屏蔽告警插件策略搜索时无法多选问题修复 --story=130827380
# Reviewed, transaction id: 73467

* fix: 优化逻辑避免竞态条件 --story=130827380
# Reviewed, transaction id: 73469

Mianhuatang8 committed:
fix: ai小鲸回填数据获取时机问题修复 --story=130202440 (#8182)

* fix: ai小鲸回填数据获取时机问题修复 --story=130202440
# Reviewed, transaction id: 73494

* fix: 模板和任务code无需脚本检查 --story=130202440

* fix: 模板和任务code无需脚本检查 --story=130202440
# Reviewed, transaction id: 73503

* fix: 删除重复初始化 --story=130202440

dengyh committed:
refactor: 修复内置插件span名称不对的问题 & span层级问题修复 --story=130572156

dengyh committed:
Merge pull request #8184 from dengyh/feat/plugin_trace_v2

refactor: 修复内置插件span名称不对的问题 & span层级问题修复 --story=130572156

Mianhuatang8 committed:
fix: 正则校验配置导致节点无法保存 --story=131004484
# Reviewed, transaction id: 73712

Mianhuatang8 committed:
fix: 节点输出参数勾选的全局变量支持手动删除  --story=130415920 (#8188)

* fix: 节点输出参数勾选的全局变量支持手动删除  --story=130415920
# Reviewed, transaction id: 73667

* fix: 节点输出参数勾选的全局变量支持手动删除  --story=130415920
# Reviewed, transaction id: 73707

Mianhuatang8 committed:
fix: 校验规则补充字符串判断 --story=131004484
# Reviewed, transaction id: 73743

dengyh committed:
refactor: 修复部分插件没有继承BasePluginService的问题 & 修复部分插件execute或者schedule返回None导致认为插件执行失败的问题 --story=130572156

dengyh committed:
refactor: 修复execute和schedule的父span不对的问题 --story=130572156

dengyh committed:
Merge pull request #8196 from dengyh/feat/plugin_trace_v3

Feat/plugin trace v3

dengyh committed:
refactor: approve插件补充调用finish_schedule --story=130572156

dengyh committed:
Merge pull request #8197 from dengyh/feat/plugin_trace_v3

refactor: approve插件补充调用finish_schedule --story=130572156

dengyh committed:
refactor: 调整span属性的内容 --story=130572156

dengyh committed:
Merge pull request #8198 from dengyh/feat/plugin_trace_v3

refactor: 调整span属性的内容 --story=130572156

dengyh committed:
refactor: 流程结束后, 流程output清理span相关的内置属性 --story=130572156

dengyh committed:
Merge pull request #8203 from dengyh/feat/plugin_trace_v3

refactor: 流程结束后, 流程output清理span相关的内置属性 --story=130572156

Commit:
feat: 增加AI分析报告企业微信推送功能 --story=131238327

Commit:
feat: 添加获取JOB节点的执行日志的内部接口 --story=131632907

dengyh committed:
Merge pull request #8202 from kemafd1sf/feat/add_ai_analysis_notify

Feat/add ai analysis notify

Mianhuatang8 committed:
fix: 节点详情预览图中状态显示错误问题修复 --story=131742471 (#8210)

* fix: 节点详情预览图中状态显示错误问题修复 --story=131742471
# Reviewed, transaction id: 74483

* fix: 节点详情预览图中状态显示错误问题修复 --story=131742471
# Reviewed, transaction id: 74489

Mianhuatang8 committed:
feat: 流程基础信息增加AI分析 --story=131224849 (#8206)

* feat: 流程基础信息增加AI分析 --story=131224849

* feat: 流程基础信息增加AI分析 --story=131224849
# Reviewed, transaction id: 74201

* fix: 接口字段修改 --story=131224849
# Reviewed, transaction id: 74544

* fix: 兼容性处理 --story=131224849
# Reviewed, transaction id: 74549

* fix: 兼容性处理 --story=131224849
# Reviewed, transaction id: 74550

Commit:
fix: 修改AI通知开关 --story=131238327

dengyh committed:
Merge pull request #8211 from kemafd1sf/feat/add_ai_analysis_notify

fix: 修改AI通知开关 --story=131238327

Commit:
fix: 修改AI通知任务task_id获取逻辑 --story=131238327

dengyh committed:
Merge pull request #8212 from kemafd1sf/feat/add_ai_analysis_notify

fix: 修改AI通知任务task_id获取逻辑 --story=131238327

dengyh committed:
Merge pull request #8209 from kemafd1sf/feat/add_inner_api_for_get_job_log

feat: 添加获取JOB节点的执行日志的内部接口 --story=131632907

Commit:
refactor: ai通知逻辑完善日志 --story=131238327

dengyh committed:
Merge pull request #8213 from kemafd1sf/feat/add_ai_analysis_notify

refactor: ai通知逻辑完善日志 --story=131238327

Commit:
fix: 添加AI通知独立worker配置 --story=131238327

dengyh committed:
Merge pull request #8214 from kemafd1sf/feat/add_ai_analysis_notify

fix: 添加AI通知独立worker配置 --story=131238327

kemafd1sf committed:
fix: 添加ai错误分析报告内容过长截断逻辑 --story=131238327

kemafd1sf committed:
fix: 企业微信个人通知修改为仅支持通知执行人 --story=131238327

Mianhuatang8 committed:
feat: AI分析设计界面重构 --story=131224849 (#8216)

* feat: AI分析设计界面重构 --story=131224849
# Reviewed, transaction id: 74883

* fix: 逻辑优化 --story=131224849
# Reviewed, transaction id: 74888

* fix: 减少重复请求 --story=131224849
# Reviewed, transaction id: 74897

* fix: 逻辑优化 --story=131224849
# Reviewed, transaction id: 74898

dengyh committed:
docs: add design doc for get_template_info unfold_subprocess parameter

新增设计文档，说明在 get_template_info 接口中增加 unfold_subprocess
参数的方案设计，用于一次性返回包含所有子流程完整配置的流程树结构。

Made-with: Cursor

dengyh committed:
docs: add implementation plan for get_template_info unfold_subprocess

详细实现计划，包含 5 个任务：serializer 新增字段、replace_template_id_recursive
工具函数、format_template_data 分支、视图层异常处理、文档更新。

Made-with: Cursor

dengyh committed:
feat: add unfold_subprocess field to IncludeTemplateSerializer

Made-with: Cursor

dengyh committed:
feat: add replace_template_id_recursive utility function

Adds recursive ID conversion for pipeline trees that contain unfolded
subprocess nodes (act["pipeline"]), where the existing replace_template_id
only processes the top-level activities.

Made-with: Cursor

dengyh committed:
Merge pull request #8215 from kemafd1sf/feat/add_ai_analysis_notify

fix: 添加ai错误分析报告内容过长截断逻辑和企业微信个人通知目前仅支持任务执行人 --story=131238327

dengyh committed:
fix: use COMMON constant instead of magic string in replace_template_id_recursive

Made-with: Cursor

dengyh committed:
feat: support unfold_subprocess in format_template_data

Made-with: Cursor

dengyh committed:
fix: use safe pop(key, None) in both format_template_data branches

Made-with: Cursor

dengyh committed:
feat: get_template_info supports unfold_subprocess parameter

Reads unfold_subprocess from serializer validated data, passes it to
format_template_data, and wraps the call in a try/except so that any
exception (e.g. recursion limit from PipelineException) returns a
structured result=False response instead of a 500.

Made-with: Cursor

dengyh committed:
fix: use accurate error message in get_template_info exception handler

Replace "unfold_subprocess" with "format_template_data" in the except block
log and response message, since the handler catches any exception from
format_template_data, not just unfold_subprocess errors. Update the
corresponding test assertion and add a code field assertion.

Made-with: Cursor

dengyh committed:
docs: update get_template_info docs and skill with unfold_subprocess parameter

Made-with: Cursor

dengyh committed:
Revert "docs: update get_template_info docs and skill with unfold_subprocess parameter"

This reverts commit a432e71e19e6c6d560791c201bae4d17ef76970f.

dengyh committed:
Merge pull request #8217 from dengyh/feat/get-template-info-unfold-subprocess

feat: get_template_info 接口新增 unfold_subprocess 参数，支持递归展开子流程配置

dengyh committed:
docs: add unfold_subprocess parameter to get_template_info API docs

Made-with: Cursor

kemafd1sf committed:
fix: ai报告通知截断兜底逻辑长度限制由字符修改为字节 --story=131238327

dengyh committed:
Merge pull request #8219 from kemafd1sf/feat/add_ai_analysis_notify

fix: ai报告通知截断兜底逻辑长度限制由字符修改为字节 --story=131238327

dengyh committed:
Merge pull request #8218 from dengyh/feat/get-template-info-unfold-subprocess

docs: add unfold_subprocess parameter to get_template_info API docs

Mianhuatang8 committed:
fix: 任务详情头部样式问题修复 --story=132219547 (#8220)

* fix: 任务详情头部样式问题修复 --story=132219547
# Reviewed, transaction id: 75236

* fix: 删除冗余样式 --story=132219547
# Reviewed, transaction id: 75237

dengyh committed:
fix: 周期任务开关操作时刷新更新人 --bug=132378020

Made-with: Cursor

RGardenia committed:
feat: 智能体流程编排 --story=131050991

dengyh committed:
feat: Job 插件支持节点名+时间戳作为 job 任务名 --story=132389598

- 新增 get_job_task_name 工具函数
- 快速执行脚本 v2.1、快速分发文件 v3.1、执行作业 v2.1
- 跨业务快速执行脚本 v1.2、跨业务快速分发文件 v1.2、跨业务执行作业 v1.2

Made-with: Cursor

dengyh committed:
refactor: 优化测试用例 --story=132331359

dengyh committed:
Merge pull request #8227 from dengyh/refactor/unittest_optimize

refactor: 优化测试用例 --story=132331359

kemafd1sf committed:
feat: 支持多种JOB节点查询日志 --story=131632907

dengyh committed:
Merge pull request #8223 from dengyh/fix/periodic-task-editor-update

fix: 周期任务开关操作时刷新更新人 --bug=132378020

dengyh committed:
refactor: 移除 all_biz 插件的节点任务名支持，仅保留 3 个核心插件 --story=132389598

Made-with: Cursor

dengyh committed:
Merge pull request #8226 from dengyh/feature/job-task-name-node

支持使用节点名称来作为job插件节点的任务名称

RGardenia committed:
fix: 流程编排鉴权 --story=131050991

dengyh committed:
Merge pull request #8228 from kemafd1sf/feat/add_all_job_plugin_node_get_job_log

feat: 支持多种JOB节点查询日志 --story=131632907

dengyh committed:
Merge pull request #8225 from RGardenia/master

feat: 智能体流程编排 --story=131050991

dengyh committed:
refactor: 优化部分内部apigw url结构 --story=132544238

dengyh committed:
Merge pull request #8229 from dengyh/refactor/optimize_apigw_configs

refactor: 优化部分内部apigw url结构 --story=132544238

dengyh committed:
fix: 修复pipeline plugin import 的问题 --bug=132544238

dengyh committed:
Merge pull request #8230 from dengyh/fix/pipeline_plugins_import

fix: 修复pipeline plugin import 的问题 --bug=132544238

dengyh committed:
refactor: 完善node_callback 文档 --story=132544238

dengyh committed:
Merge pull request #8231 from dengyh/refactor/optimize_callback_docs

refactor: 完善node_callback 文档 --story=132544238

dengyh committed:
refactor: 修复job新插件版本导致的import问题 --story=132389598

dengyh committed:
Merge pull request #8234 from dengyh/fix/job_plugin_import

refactor: 修复job新插件版本导致的import问题 --story=132389598

Mianhuatang8 committed:
fix: ai小鲸icon功能支持开关  --story=132588811 (#8233)

* fix: ai小鲸icon功能支持开关  --story=132588811
# Reviewed, transaction id: 75668

* fix: 开关状态统一管理 --story=132588811
# Reviewed, transaction id: 75670

* fix: 开关状态统一管理 --story=132588811
# Reviewed, transaction id: 75698

dengyh committed:
refactor: 网关接口作为mcp服务时, 返回pipelien_tree进行裁剪 --story=132636521

dengyh committed:
Merge pull request #8235 from dengyh/feat/pipeline-tree-trim

refactor: 网关接口作为mcp服务时, 返回pipelien_tree进行裁剪 --story=132636521

RGardenia committed:
fix: 流程编排 元数据默认值设置 --story=131050991 (#8236)

* fix: 流程编排 元数据默认值设置 --story=131050991

* fix: 流程编排 元数据默认值 mutations 设置 --story=131050991

* fix: 流程编排 元数据重置 --story=131050991

RGardenia committed:
feat: 创建模版网关接口 --story=131050991

Mianhuatang8 committed:
feat: 流程编辑页面增加“历史任务”的快捷入口 --story=132713061 (#8237)

* feat: 流程编辑页面增加“历史任务”的快捷入口 --story=132713061
# Reviewed, transaction id: 75838

* feat: 流程编辑页面增加“历史任务”的快捷入口 --story=132713061
# Reviewed, transaction id: 75845

RGardenia committed:
fix: 创建模版网关接口 排版 --story=131050991

kemafd1sf committed:
fix: 修复网关API返回DRF Response导致500 错误 --story=132739429

dengyh committed:
Merge pull request #8240 from kemafd1sf/fix/api_response_error

fix: 修复网关API返回DRF Response导致500 错误 --story=132739429

RGardenia committed:
docs: 创建模版网关接口文档 --story=131050991

dengyh committed:
Merge pull request #8241 from RGardenia/gardenia/create_template_api

feat: 创建模版网关接口

dengyh committed:
feat: get_template_info 网关接口支持 format=yaml 参数导出 YAML 格式 pipeline_tree --story=132767026

dengyh committed:
feat: create_template 网关接口支持 format=yaml 参数导入 YAML 格式流程 --story=132767026

Made-with: Cursor

dengyh committed:
refactor: 补充相关开发设计 --story=132767026

dengyh committed:
fix: 修复 PR review 问题：template_order 越界保护、多模板警告日志、文档补充说明 --story=132767026

Made-with: Cursor

dengyh committed:
Merge pull request #8243 from dengyh/feat/get-template-info-yaml-format

feat: get_template_info 和 create_template 网关接口支持 format=yaml 参数导出 YAML 格式 pipeline_tree…

dengyh committed:
fix: yaml文件默认不提交 --bug=132767026

RGardenia committed:
feat: 流程编排添加功能开关逻辑 (#8244)

* feat: 流程编排添加功能开关逻辑 --story=131050991

* fix: 错误码修改 --story=131050991

* feat: 流程编排添加功能开关逻辑 前端 --story=131050991
